gen/quick_test.go

```diff
@@ -944,3 +991,22 @@ func (q *quickSuite) testErrorName(t *testing.T) {
 
 	assert.NotPanics(t, func() { _ = v.ErrorName() })
 }
+
+// deepCopy copies the thriftType interface as a work-around until
+// mutation in the ToWire methods are removed.
+func deepCopy(v thriftType) thriftType {
+	c := reflect.New(reflect.TypeOf(v).Elem())
+
+	val := reflect.ValueOf(v).Elem()
```

Comment:
minor simplification: can re-use `val` to get type instead of reflecting it again.

```suggestion
	val := reflect.ValueOf(v).Elem()
	c := reflect.New(val.Type())
```